[2024-09-24T17:15:33.102+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-24T17:15:33.117+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: random_forest_classifier_dag.get_data manual__2024-09-24T17:15:29.806228+00:00 [queued]>
[2024-09-24T17:15:33.125+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: random_forest_classifier_dag.get_data manual__2024-09-24T17:15:29.806228+00:00 [queued]>
[2024-09-24T17:15:33.127+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-24T17:15:33.137+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_data> on 2024-09-24 17:15:29.806228+00:00
[2024-09-24T17:15:33.145+0000] {standard_task_runner.py:64} INFO - Started process 21604 to run task
[2024-09-24T17:15:33.148+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'random_forest_classifier_dag', 'get_data', 'manual__2024-09-24T17:15:29.806228+00:00', '--job-id', '349', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/.ipynb_checkpoints/hw1-checkpoint.py', '--cfg-path', '/tmp/tmpon9tx336']
[2024-09-24T17:15:33.150+0000] {standard_task_runner.py:91} INFO - Job 349: Subtask get_data
[2024-09-24T17:15:33.190+0000] {task_command.py:426} INFO - Running <TaskInstance: random_forest_classifier_dag.get_data manual__2024-09-24T17:15:29.806228+00:00 [running]> on host 82997fc6deb7
[2024-09-24T17:15:33.253+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='random_forest_classifier_dag' AIRFLOW_CTX_TASK_ID='get_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-24T17:15:29.806228+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-24T17:15:29.806228+00:00'
[2024-09-24T17:15:33.254+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-24T17:15:33.264+0000] {hw1-checkpoint.py:90} INFO - Get data...
[2024-09-24T17:15:33.429+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-09-24T17:15:33.430+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-09-24T17:15:33.917+0000] {hw1-checkpoint.py:108} INFO - Finish get data.
[2024-09-24T17:15:33.918+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-24T17:15:33.920+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-24T17:15:33.930+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=random_forest_classifier_dag, task_id=get_data, run_id=manual__2024-09-24T17:15:29.806228+00:00, execution_date=20240924T171529, start_date=20240924T171533, end_date=20240924T171533
[2024-09-24T17:15:33.965+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-24T17:15:33.987+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-24T17:15:33.991+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
